src/ontology/mondo-edit.obo

```diff
@@ -433276,6 +433265,26 @@ intersection_of: MONDO:0019952 {source="https://clinicalgenome.org/affiliation/4
 intersection_of: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/10483 {source="https://clinicalgenome.org/affiliation/40031/"} ! RYR1
 created_by: http://orcid.org/0000-0001-5208-3432
 
+[Term]
+id: MONDO:0100151
+name: nephropathic cystinosis
+def: "An autosomal recessive condition caused by mutation(s) in the CTNS gene, encoding cystinosin. It is a sub-type of cystinosis, in which accumulation of cystine in the kidney results in renal dysfunction." [NCIT:C129932]
+synonym: "Abderhalden Kaufmann Lignac syndrome" EXACT [MESH:C535335]
+synonym: "Abderhalden Lignac Kaufmann disease" EXACT [GARD:0010074]
+synonym: "Abderhalden-Kaufmann-Lignac syndrome" EXACT [MONDO:0021725]
+synonym: "Abderhalden-Lignac-Kaufmann disease" EXACT [MESH:C535335]
+synonym: "CTNS" RELATED [OMIM:219800]
+synonym: "cystinosis, nephropathic" EXACT [MONDO:Lexical, OMIM:219800]
+synonym: "cystinosis, nephropathic; CTNS" EXACT [OMIM:219800]
+xref: MESH:C535335 {source="MONDO:equivalentTo"}
+xref: NCIT:C129932 {source="MONDO:equivalentTo"}
+xref: OMIM:219800 {source="MONDO:equivalentTo", source="Orphanet:411629"}
+xref: UMLS:C2930877 {source="MONDO:equivalentTo", source="GARD:0010074"}
+is_a: MONDO:0016239 {source="MESH:C535335", source="NCIT:C129932"} ! cystinosis
+relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/2518 {source="OMIM:219800"} ! CTNS
+relationship: has_modifier HP:0000007 {source="NCIT:C129932"} ! Autosomal recessive inheritance
```

Comment:
NCIT is not an authoritative source for MoI. Always use OMIM, ORDO, NORD, GARD, literature, ... here

Reply:
thanks - I fixed this on https://github.com/monarch-initiative/mondo/pull/1798